lib/vrl/stdlib/Cargo.toml

```diff
@@ -53,6 +53,7 @@ default = [
     "ends_with",
     "exists",
     "flatten",
+    "float",
```

Comment:
This one was missing from the list of functions (all the other diffs are merely sorting related)